apps/dog-food/Styled.hs

```diff
@@ -19,6 +22,21 @@ plain = Styled []
 bPlain :: ByteString -> StyledString
 bPlain = Styled [] . B.unpack
 
+styleMarkdown :: Markdown -> StyledString
+styleMarkdown (s1 :|: s2) = styleMarkdown s1 <> styleMarkdown s2
+styleMarkdown (Markdown Snippet s) = plain . T.unpack $ '`' `T.cons` s `T.snoc` '`'
+styleMarkdown (Markdown f s) = Styled sgr $ T.unpack s
+  where
+    sgr = case f of
+      Bold -> [SetConsoleIntensity BoldIntensity]
+      Italic -> [SetUnderlining SingleUnderline, SetItalicized True]
+      Underline -> [SetUnderlining SingleUnderline]
+      StrikeThrough -> [SetSwapForegroundBackground True]
+      Colored Black -> [SetColor Foreground Dull Black]
```

Comment:
can't find a format/color for this.. did it stay from testing?

Reply:
yes - removing in the next PR